src/components/Projectcard Grid Home/ProjectCardGridHome.js

```diff
@@ -5,9 +5,11 @@ export default function ProjectCardGridHome() {
 	return (
 		<ul>
 			{projects.map(project => {
-				<li>
-					<ProjectCardHome name={project.name} />
-				</li>;
+				return (
+					<li key={project.id}>
+						<ProjectCardHome project={project} />
+					</li>
+				);
```

Comment:
I don't understand why your component is called "ProjectCardGridHome" if there is no Grid?

Reply:
Hey, I called it that way, because we used this naming in other exercises. But your right, it should be a list :)